src/helm/benchmark/window_services/window_service_factory.py

```diff
@@ -96,21 +112,14 @@ def get_window_service(model_name: str, service: TokenizerService) -> WindowServ
 
             window_service = OpenAIWindowService(service)
         elif organization == "AlephAlpha":
-            from helm.benchmark.window_services.luminous_window_service import (
-                LuminousBaseWindowService,
-                LuminousExtendedWindowService,
-                LuminousSupremeWindowService,
-                LuminousWorldWindowService,
-            )
-
             if engine == "luminous-base":
-                window_service = LuminousBaseWindowService(service)
+                window_service = DefaultWindowService(service, "AlephAlpha/luminous-base", max_sequence_length=2048)
             elif engine == "luminous-extended":
-                window_service = LuminousExtendedWindowService(service)
+                window_service = DefaultWindowService(service, "AlephAlpha/luminous-extended", max_sequence_length=2048)
             elif engine == "luminous-supreme":
-                window_service = LuminousSupremeWindowService(service)
+                window_service = DefaultWindowService(service, "AlephAlpha/luminous-supreme", max_sequence_length=2048)
             elif engine == "luminous-world":
-                window_service = LuminousWorldWindowService(service)
+                window_service = DefaultWindowService(service, "AlephAlpha/luminous-world", max_sequence_length=2048)
```

Comment:
Now that we have quite a complete `DefaultWindowService` why don't we remove the others? I rapidly looked into it and we can remove more window services than just luminous

Reply:
Because I didn't have time to do it =p

We definitely should.